backend/Application/DTOs/Admin/RegisterRequest.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Application.DTOs.Admin
+{
+    public class RegisterRequest
+    {
+        public string? Email { get; set; }
+
+        public string? Username { get; set; }
+
+        public string? Password { get; set; }
+
+        public bool EmailConfirmed { get; set; }
+
+        public string? Phone {  get; set; }
```

Comment:
Let's don't use Phone for the Register dto.

Reply:
Done!